### PR TITLE
Ember 1.10.0

### PR DESCRIPTION
**Pull request by cdmwebs**
This dumps the canary ref and uses 1.10.0. All tests pass and it seems to work as expected. Would appreciate some additional :eyes: @mitchlloyd @rockwood.

### CODE REVIEW

File: client/app/controllers/chapter.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ var alias = Ember.computed.alias;
 export default Ember.ObjectController.extend({
   needs: ['application', 'chapters', 'course'],
   title: alias('controllers.application.title'),
+  course: alias('model.section.course'),
```
Comment (cdmwebs) [inline client/app/controllers/chapter.js]: @mitchlloyd is there a better way here? I'm still a little confused as to how best to reference these associations.

Comment (mitchlloyd) [inline client/app/controllers/chapter.js]: This seems fine. This alias could also go in the chapter model. If we think it doesn't make sense to set the course on a chapter we could use a computed.readOnly to be more restrictive.

**Comment by cdmwebs** [comment]
Looks like it @mitchlloyd!

**Comment by cdmwebs** [comment]
Merged. Thanks @mitchlloyd!

**Pull request merged**